src/components/NonProfitApply/NonProfitApply.js

```diff
@@ -79,7 +104,17 @@ export default function NonProfitApply() {
     keyStaffAvailability: [],
   });
 
+  const [ip, setIP] = useState("");
+
+  const getIP = async () => {
+    const res = await axios.get("https://api.ipify.org/?format=json");
```

Comment:
Not sure if this is the best way to get an IP address, but it worked really well during testing.  You can always fallback to X_FORWARDED_FOR in the Header, but that can be spoofed